devlib/platform/arm.py

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
 #
 from __future__ import division
+import csv
```

Comment:
Just looking over these commits and noticed there's some residual diff-noise in this file.